src/Model/IO/IOStyle.php

```diff
@@ -4,13 +4,146 @@
 
 use Symfony\Component\Console\Style\SymfonyStyle;
 use Symfony\Component\Console\Style\StyleInterface;
+use Symfony\Component\Process\Process;
+use Waffle\Model\Output\Runner;
 
 class IOStyle extends SymfonyStyle implements StyleInterface
 {
-    // Intentionally blank for now. We can use this class to add any new helper
-    // methods we want other than what is provided by SymfonyStyle. Further, we
-    // can also override SymfonyStyle. Ideally, all IO wlil be routed through
-    // this class so that we can have consistent output styles.
-
     // See https://github.com/symfony/console/blob/5.x/Style/SymfonyStyle.php.
+    
+    /**
+     *
+     * @param $message
+     * @param $command
+     *
+     * @return string
+     */
+    public function message($message, $command)
```

Comment:
This is my bad -- when I stopped working on the refactor, this is where I left off since I knew it was going to break the updated code.

I would like to keep this file specifically for output so that we can define our own style of sorts. This would also be the place to handle any custom output that we want for something that is not supported out of the box (ie, tables, progress bars, etc...).

Instead of putting the stuff from `Runner` here, can you move them to a `CliHelper` class instead? I think that would be more intuitive in long term for these cases where we have a `Process` that we need to setup or do some funny business to get the output that we need. From there, then we can use the `IO` class to display the data to the user.

Reply:
Sure, that works. Where should `CliHelper` live and how should it be instantiated in the update tool? Should we put it in `BaseCommand`?

Reply:
I think it would fit in fine either at `Waffle\Helper\CliHelper` or `Waffle\Model\Cli\CliHelper`, I don't really have any preference there.